internal/store/postgres/asset_repository.go

```diff
@@ -540,33 +696,14 @@ func (r *AssetRepository) update(ctx context.Context, assetID string, newAsset,
 		}
 		newAsset.Version = newVersion
 		newAsset.ID = oldAsset.ID
-		newAsset.UpdatedAt = time.Now()
-
-		query, args, err := sq.Update("assets").
-			Set("urn", newAsset.URN).
-			Set("type", newAsset.Type).
-			Set("service", newAsset.Service).
-			Set("name", newAsset.Name).
-			Set("description", newAsset.Description).
-			Set("data", newAsset.Data).
-			Set("url", newAsset.URL).
-			Set("labels", newAsset.Labels).
-			Set("updated_at", newAsset.UpdatedAt).
-			Set("updated_by", newAsset.UpdatedBy.ID).
-			Set("version", newAsset.Version).
-			Where(sq.Eq{"id": assetID}).
-			PlaceholderFormat(sq.Dollar).
-			ToSql()
-		if err != nil {
-			return fmt.Errorf("build query: %w", err)
-		}
+		newAsset.UpdatedAt = newAsset.RefreshedAt // current time
```

Comment:
explained in our discussion, as to equalize the current time between UpdatedAt and RefreshedAt (or CreatedAt and RefreshedAt in insert), and because Upsert can to be update OR insert, while the RefreshedAt updated to be current time in Service layer